string-manipulation-2/src/first-chars.js

```diff
@@ -0,0 +1,23 @@
+/* exported firstChars */
+
+/*
+- create storage for output
+- look at each character one by one
+- if the string length is greater than the length parameter
+  - add each character to the output
+- otherwise
+  - the output will equal the string
+- return output
+*/
+
+function firstChars(length, string) {
+  var output = '';
+  for (var i = 0; i < length; i++) {
+    if (string.length > length) {
+      output += string[i];
+    } else {
+      output = string;
+    }
```

Comment:
I would make this a conditional above the for loop, and you can simply `return` the string there so that this loop doesn't need to run if it isn't necessary.